Core/EntityManager/src/test/java/eu/neclab/ngsildbroker/entityhandler/controller/EntityBatchControllerTest.java

```diff
@@ -1,263 +1,380 @@
+  
+  @SpringBootTest(properties=
+  {"spring.main.allow-bean-definition-overriding=true"})
+  
+  @RunWith(SpringRunner.class)
+  
+  @AutoConfigureMockMvc//(secure = false) public class
+  public class EntityBatchControllerTest {
+  
+  @Autowired 
+  private MockMvc mockMvc;
+  
+  @MockBean 
+  private EntityService entityService;
+  
+  private String payload; 
+  private String deletePayload;
+  
+  public static boolean
+  checkEntity = false;
+  
+  @Before
+  public void setup() throws Exception { //@formatter:off
+  
+				  payload= "[  \r\n" + "{  \r\n" +
+							  "   \"id\":\"urn:ngsi-ld:Vehicle:A101\",\r\n" +
+							  "   \"type\":\"Vehicle\",\r\n" + "   \"brandName\":\r\n" + "      {  \r\n" +
+							  "         \"type\":\"Property\",\r\n" + "         \"value\":\"Mercedes\"\r\n"
+							  + "      },\r\n" + "   \"speed\":{  \r\n" +
+							  "         \"type\":\"Property\",\r\n" + "         \"value\":80\r\n" +
+							  "    }\r\n" + "},\r\n" + "{  \r\n" +
+							  "   \"id\":\"urn:ngsi-ld:Vehicle:A102\",\r\n" +
+							  "   \"type\":\"Vehicle\",\r\n" + "   \"brandName\":\r\n" + "      {  \r\n" +
+							  "         \"type\":\"Property\",\r\n" + "         \"value\":\"Mercedes\"\r\n"
+							  + "      },\r\n" + "   \"speed\":{  \r\n" +
+							  "         \"type\":\"Property\",\r\n" + "         \"value\":81\r\n" +
+							  "    }\r\n" + "},\r\n" + "{  \r\n" +
+							  "   \"id\":\"urn:ngsi-ld:Vehicle:A103\",\r\n" +
+							  "   \"type\":\"Vehicle\",\r\n" + "   \"brandName\":\r\n" + "      {  \r\n" +
+							  "         \"type\":\"Property\",\r\n" + "         \"value\":\"Mercedes\"\r\n"
+							  + "      },\r\n" + "   \"speed\":{  \r\n" +
+							  "         \"type\":\"Property\",\r\n" + "         \"value\":82\r\n" +
+							  "    }\r\n" + "}\r\n" + "]";
+							  
+			  deletePayload = "[  \r\n" + "\"urn:ngsi-ld:Vehicle:A101\",\r\n" +
+							    "\"urn:ngsi-ld:Vehicle:A102\"\r\n" + "]"; 
+  
+			
+			  
+			  //@formatter:on 
+  
+  
+  }
+  
+  @After
+  public void tearDown(){ 
+	  payload = null; 
+	  deletePayload = null; 
+	  
+    }
+ // setup close
+  
+ /**
 	 * this method is use for create the multiple entity
 	 */
-/*
- * @Test public void createMultipleEntityTest() { try {
- * 
- * BatchResult batchResult = new BatchResult();
- * batchResult.addSuccess("urn:ngsi-ld:Vehicle:A101");
- * batchResult.addSuccess("urn:ngsi-ld:Vehicle:A102");
- * batchResult.addSuccess("urn:ngsi-ld:Vehicle:A103");
- * when(entityService.createMultipleMessage(any(),any())).thenReturn(batchResult
- * ); mockMvc.perform(post("/ngsi-ld/v1/entityOperations/create").contentType(
- * AppConstants.NGB_APPLICATION_JSON)
- * .accept(AppConstants.NGB_APPLICATION_JSONLD).content(payload))
- * .andExpect(status().isCreated()); verify(entityService,
- * times(1)).createMultipleMessage(any(),any()); } catch (Exception e) {
- * Assert.fail(); e.printStackTrace(); } }
- * 
- *//**
+
+	@Test
```

Comment:
[Minor] Add doc/comment in each test case like as what is the purpose of test case same as controller classes.